### PR TITLE
Validate Level Roles can be Managed

### LINKED ISSUE #1581: [FEAT] - Validate Level Roles can be managed
### What feature would you like to see?

In the settings permission validation, Becca should not only check that she has manage roles permissions, but that a set level role can be managed by her.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Vyvy-vi**
# Pull Request

## Description

This PR adds a check to the `/levels set level_role: Role` command, which warns user if the mentioned role can't be managed by becca. Bumped up puppeteer as older version was incompatible

## Related Issue

Closes #1581 

## Documentation

For _any_ version updates, please verify if the [documentation page](https://docs.beccalyria.com?utm_source=github&utm_medium=pr-template) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by Vyvy-vi** [comment]
Screenshot 2023-05-24 at 7 13 00 PM

**Comment by naomi-lgbt** [comment]
That's just a deprecation warning, not an error. We shouldn't install puppeteer as a separate dependency - the package that is actually using it should update their version instead.

File: src/commands/subcommands/config/handleSet.ts [hunk 1]
```diff
@@ -69,6 +69,19 @@ export const handleSet: SettingsHandler = async (
       }
     }
 
+    if (
+      setting === "level_roles" &&
+      interaction.options.getRole("role", true).position <
```
Comment (naomi-lgbt) [inline src/commands/subcommands/config/handleSet.ts:74]:
```suggestion
      interaction.options.getRole("role", true).position >
```
Think this needs to be inverted. The everyone role is at the bottom of the list and has a position of 0. The higher the role in the list, the higher the position.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: src/commands/subcommands/config/handleSet.ts (hunk 1 above)
Comment (Vyvy-vi) [reply src/commands/subcommands/config/handleSet.ts:74]: oh

**Review by naomi-lgbt — APPROVED** [review]
Okie dokie

**Comment by naomi-lgbt** [comment]
## Thank You

Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**